.vscode/settings.json

```diff
@@ -19,5 +19,6 @@
   },
   "[prisma]": {
     "editor.defaultFormatter": "Prisma.prisma"
-  }
+  },
+  "prettier.configPath": "packages/backend/.prettierrc"
```

Comment:
That might not be the best solution, we might want different settings for the frontend (e. g., in react projects , we usually  use double quotes in JSX (like name="Samu"), but single quote everywhere else). But we can figure this out some other time

Reply:
Prettier was looking red without a config, and this seemed the easy solution, but i agree, that we should do something about this.